package.json

```diff
@@ -8,8 +8,8 @@
     "socket",
     "socket.io"
   ],
```

Comment:
Bonus points if you add a test that uses TypeScript, or convert an existing test to TypeScript :)

Reply:
I'll take a look, could be good to validate typing

Reply:
I wrote the tests in TypeScript, and they passed! 
After noticing that ecstatic lacked typings and was marked as deprecated, I removed it and manually wrote the code for the static file routing.
https://github.com/sffc/socketio-file-upload/pull/144/commits/5042533a37e48ad0fa8107e6cbc49e923689342e